### DISCUSSION TITLE
Sign in with Apple doesn't work in android (capacitor-firebase/authentication: 5.0.0)

### DISCUSSION THREAD

**Discussion by soulcheck** [capawesome-team/capacitor-firebase#386] [Q&A]
Hi. 

I'm having a weird problem where trying to sign in with apple results in a custom Chrome tab (used for sign in) opening and immediately closing, as shown below 

https://github.com/capawesome-team/capacitor-firebase/assets/424486/ecb0df9c-57c8-4010-a7c6-520f06fca1f3

This happens both in emulator and on device (with prod install).

Sign in with google works without problems, which makes me think checksums are correct in firebase auth config. 

Similarly, apple sign in works on web. So the service configuration on apple side is is fine.

I even managed to grab the activity url and paste it into chrome in emulator and it opened fine.

Not sure what can be causing this behavior, or even how do debug it.

Any help or ideas will be greatly appreciated!

**Comment by robingenz** [comment]
Sorry for my late response.
I don't assume it's a bug, since it works in the [demo app](https://github.com/robingenz/capacitor-firebase-authentication-demo).
So it should be a configuration issue. Can you post the logcat output?
